types/markdown-it/lib/renderer.d.mts

```diff
@@ -17,6 +17,11 @@ export interface RenderRuleRecord {
 }
 
 export default class Renderer {
+    /**
+     * Creates new {@link Renderer} instance and fill {@link Renderer#rules} with defaults.
+     */
+    constructor();
```

Comment:
this class is new-able -- https://unpkg.com/browse/markdown-it@14.1.0/lib/renderer.mjs (see comment starting on line 109)